README.md

```diff
@@ -49,6 +49,14 @@ To install this, simply run:
 pkgin install repofetch
 ```
 
+#### Via AUR
+
+If you are using an Arch machine, you can install repofetch from the [Aur](https://aur.archlinux.org).
+
+```
+yay -S ruby-repofetch
+```
+
```

Comment:
This would install version `>= 0.4.0`, so this installation instruction should be above this line: https://github.com/spenserblack/repofetch/blob/b32afbeb30363241dac86ea38a2858890805499a/README.md?plain=1#L32

Reply:
@spenserblack done.